packages/inter-protocol/test/provisionPool.test.js

```diff
@@ -395,7 +397,9 @@ test('makeBridgeProvisionTool handles duplicate requests', async t => {
   await handler.fromBridge({
     type: 'PLEASE_PROVISION',
     address,
+    nickname: 'test',
     powerFlags: PowerFlags.SMART_WALLET,
+    submitter: address,
   });
```

Comment:
The old non-Array `powerFlags` string was incorrect, and fixing it might actually cause this test to break.  I hope not.

Also, the required `blockHeight` and `blockTime` values (part of ActionHeader) may need adjusting.

```suggestion
  await handler.fromBridge({
    type: 'PLEASE_PROVISION',
    blockHeight: '123',
    blockTime: '1737432662',
    address,
    nickname: 'test',
    powerFlags: [PowerFlags.SMART_WALLET],
    submitter: address,
  });
```

Reply:
>The old non-Array powerFlags string was incorrect, and fixing it might actually cause this test to break. I hope not.

It didn't because `.includes` is available on both Array and String and happens to be equivalent in this case. I made a comment about this instead of introducing another runtime change to validate array. I figure the Golang ensures that for real bridge messages so it should be safe but I didn't want to further mix runtime changes in with this types PR.